### ISSUE TITLE
auto completion fails

### ISSUE THREAD

**Issue by wonderful-panda** [Quramy/tsuquyomi#99]
Sometimes tsuquyomi fails to show candidates.

For investigation, Ｉ added 2 `echom` to `tsuquyomi#tsClient#sendRequest`

```vim
function! tsuquyomi#tsClient#sendRequest(line, delay, retry_count, response_length)
  "call s:debugLog('called! '.a:line)
  call tsuquyomi#tsClient#startTss()
  call s:P.writeln(s:tsq, a:line)

""""" FIRST ONE
  echom '[REQUEST] ' . a:line

  let l:retry = 0
  let response_list = []

  while len(response_list) < a:response_length

    """ (snip)

    if type == 'matched'

      """ (snip)

      for res_item in l:res_list
        " ignore 2nd response of reload command #62
        if res_item !~'{"reloadFinished":true}}$'
""""" SECOND ONE
          echom '[RESPONSE] ' . res_item
          call add(response_list, res_item)
        endif
      endfor
    else
      echom '[Tsuquyomi] TSServer request was timeout:'.a:line
      return response_list
    endif

  endwhile
  "call s:debugLog(a:response_length.', '.len(response_list))
  return response_list
endfunction

```

And I got this result.

```
[REQUEST] {"seq":0,"arguments":{"file":".../index.ts","tmpfile":".../Temp/VIA2D61.tmp"},"type":"request","command":"reload"}
[RESPONSE] {"seq":0,"type":"event","event":"configFileDiag","body":{"configFile":".../tsconfig.json","diagnostics":[]}}
[RESPONSE] {"seq":0,"type":"event","event":"configFileDiag","body":{"configFile":".../tsconfig.json","diagnostics":[]}}
[REQUEST] {"seq":0,"arguments":{"file":".../index.ts","offset":19,"line":24,"prefix":""},"type":"request","command":"completions"}
[RESPONSE] {"seq":0,"type":"response","command":"reload","request_seq":0,"success":true}
```

When sending `reload` request, `tsserver` outputs `configFileDiag` event first, and then outputs `reload` response.
Tsuquyomi captures `configFileDiag` event as a `reload` response, and captures `reload` response as a  `completions` response.

**Comment by Quramy** [comment]
@wonderful-panda Thanks for your report, I'll fix it!

**Comment by wonderful-panda** [comment]
My TypeScript version is 2.0.8.
This problem seems to be in 2.0.6 or later.
Thank you.

**Comment by Quramy** [comment]
@wonderful-panda I reproduced this issue and fix it at the latest commit(2527074b2d7c7ffdf62c868c1ba050d5fa0b24eb). Please check out it.

**Comment by wonderful-panda** [comment]
Works fine. Thank you !
